docs/syntax-support.rst

```diff
@@ -469,3 +469,74 @@ The GROUPING(col_a, col_b) results in _col3 and it represents a bit set converte
+                - CrossJoin => [expr:integer, min:integer, count_all:bigint, count_non_null:bigint]                                                                    >
+                        Distribution: REPLICATED                                                                                                                       >
+                    - Project[projectLocality = LOCAL] => [expr:integer]                                                                                               >
+                            expr := INTEGER'19'                                                                                                                        >
+                        - LocalExchange[ROUND_ROBIN] () => []                                                                                                          >
+                            - Values => []                                                                                                                             >
+                                        ()                                                                                                                                 >
+                        - Aggregate => [min:integer, count_all:bigint, count_non_null:bigint]                                                                              >
+                                min := "presto.default.min"((field)) (1:17)                                                                                                >
+                                count_all := "presto.default.count"(*) (1:17)                                                                                              >
+                                count_non_null := "presto.default.count"((field)) (1:17)                                                                                   >
+                            - Values => [field:integer]                                                                                                                    >
+                                        (INTEGER'19')                                                                                                                          >
+                                        (INTEGER'20')                                                                                                                          >
+                                        (INTEGER'21')           
+
+For **<, <=, >, >=**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The only two changes are:
+
+1. the aggregate function to get the max or min value.
+
+2. the compare operator in the join step.
+
+For **=**
+^^^^^^^^^^
+There will be two aggregate functions **MIN** and **MAX**, and the join condition will be **(min) = (max)) AND ((expr) = (max))**.
```

Comment:
`(min) = (max)) AND ((expr) = (max))` is not join condition but project expressions.